scripts/cadical-1.4.1-patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/scripts/make-build-header.sh b/scripts/make-build-header.sh
+index e8f6746..1290024 100755
+--- a/scripts/make-build-header.sh
++++ b/scripts/make-build-header.sh
+@@ -20,7 +20,7 @@ warning () {
+ 
+ #--------------------------------------------------------------------------#
+ 
+-[ ! -f VERSION -a ! -f ../VERSION ] && \
++[ ! -f VERSION.txt -a ! -f ../VERSION.txt ] && \
+ die "needs to be called from build sub-directory"
+ 
+ [ -f makefile ] || \
+@@ -29,7 +29,7 @@ warning "could not find 'makefile'"
+ #--------------------------------------------------------------------------#
+ # The version.
+ #
+-VERSION="`cat ../VERSION`"
++VERSION="`cat ../VERSION.txt`"
```

Comment:
Ummm... why the rename?

Reply:
I added a comment [here](https://github.com/diffblue/cbmc/pull/7493/files#diff-3e2513390df543315686d7c85bd901ca9256268970032298815d2f893a9f0685R182) that explains why the rename is necessary. In short, the build failed on macos-11 without this change:

https://github.com/diffblue/cbmc/actions/runs/4138376169/jobs/7154711888#step:9:782

because the used macos-11 image is case insensitive, and so it ends up using the `VERSION` file in cadical's root directory for an `#include<version>` directive, as opposed to using the system's `version` header file. See this thread for more details:

https://github.com/marian-nmt/marian-dev/issues/871#issuecomment-891301114